x/evmutil/keeper/evm.go

```diff
@@ -48,7 +48,7 @@ func (k Keeper) CallEVM(
 		)
 	}
 
-	resp, err := k.CallEVMWithData(ctx, from, &contract, data)
+	resp, err := k.CallEVMWithData(ctx, from, &contract, data, big.NewInt(0))
```

Comment:
here we hardcode this to 0 because this evm call doesn't involve sending any funds but just triggering some action on the contract?